src/crawlee/configuration.py

```diff
@@ -51,14 +51,14 @@ class Configuration(BaseSettings):
     ] = False
 
     log_level: Annotated[
-        int,
+        Literal['DEBUG', 'INFO', 'WARNING', 'ERROR'],
```

Comment:
`NOTSET`? `CRITICAL`?

Reply:
These are what Javascript crawlee supports. Notset doesn't sound like a great idea to me here, but I'm open to adding `CRITICAL`.